examples/tf/rl2_ppo_ml10.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+"""Example script to run RL2 in ML10."""
+from metaworld.benchmarks import ML10
+
+from garage import wrap_experiment
+from garage.experiment import task_sampler
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearFeatureBaseline
+from garage.sampler import LocalSampler
+from garage.tf.algos import RL2PPO
+from garage.tf.algos.rl2 import RL2Env
+from garage.tf.algos.rl2 import RL2Worker
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianGRUPolicy
+
+
+@wrap_experiment
+def rl2_ppo_ml10(ctxt=None, seed=1):
+    """Train PPO with ML10 environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
+    """
+    set_seed(seed)
+    with LocalTFRunner(snapshot_config=ctxt) as runner:
+        max_path_length = 100
```

Comment:
can you make these args of the experiment function instead? they can still have defaults.

Reply:
fixed